data/images/recordsheets/submarine_dualturret_standard.svg

```diff
@@ -1291,21 +1289,21 @@
+                                                                          ><text x="6.000" y="20.000" fill="#231f20" text-anchor="start" style="font-family:Eurostile;font-size:4.800px;font-weight:normal;font-style:normal" textLength="138.399" lengthAdjust="spacingAndGlyphs"
                                                                           >Naval Combat Vehicle Critical Hits Table below (see Combat, p. 192, in </text
-                                                                          ><text x="143.431" y="20.000" fill="#231f20" text-anchor="start" style="font-family:Eurostile;font-size:4.800px;font-weight:normal;font-style:italic" textLength="28.690" lengthAdjust="spacingAndGlyphs"
-                                                                          >Total Warfare </text
-                                                                          ><text x="172.121" y="20.000" fill="#231f20" text-anchor="start" style="font-family:Eurostile;font-size:4.800px;font-weight:normal;font-style:normal" textLength="1.171" lengthAdjust="spacingAndGlyphs"
+                                                                          ><text x="144.399" y="20.000" fill="#231f20" text-anchor="start" style="font-family:Eurostile;font-size:4.800px;font-weight:normal;font-style:italic" textLength="30.071" lengthAdjust="spacingAndGlyphs"
+                                                                          >Total Warfare</text
+                                                                          ><text x="174.470" y="20.000" fill="#231f20" text-anchor="start" style="font-family:Eurostile;font-size:4.800px;font-weight:normal;font-style:normal" textLength="1.179" lengthAdjust="spacingAndGlyphs"
                                                                           > </text
                                                                           ><text fill="#231f20" x="6.000" y="25.000" style="font-family:Eurostile;font-size:4.800px;font-weight:normal;font-style:normal" text-anchor="start"
                                                                           >for more information).</text
                                                                           ><text x="6.000" y="30.000" fill="#231f20" text-anchor="start" style="font-family:Eurostile;font-size:4.800px;font-weight:normal;font-style:normal" textLength="199.000" lengthAdjust="spacingAndGlyphs"
-                                                                          >A result of 12 on the Naval Combat Vehicles Hit Location Table may infict critical hit</text
+                                                                          >A result of 12 on the Naval Combat Vehicles Hit Location Table may inflict critical hit</text
```

Comment:
Maybe this should be may inflict a critical hit?

Reply:
Odd that didn't come up in my search for infict.

Reply:
Oh, maybe I should read better.